index.d.ts

```diff
@@ -353,7 +353,7 @@ export interface TimeoutFn {
 	 * Set a timeout for the test, in milliseconds. The test will fail if the timeout is exceeded.
 	 * The timeout is reset each time an assertion is made.
```

Comment:
The doc comment should mention the `message` parameter and what it's useful for.

Reply:
Updated docs. Two questions though:

-  Most of existing type definitions don't document all arguments. Should we consider updating them?
- Are these regular jsdoc comments? For now I added the new argument documentation in a new line, but if jsdocs can be used I can also update the documentation to use that format.